stag-library-compiler/src/main/java/com/vimeo/stag/processor/generators/AdapterGenerator.java

```diff
@@ -58,16 +57,26 @@ static String getJsonName(@NotNull Element element) {
     }
 
     /**
-     * Returns the alternate name for the {@link Element}
+     * Returns the alternate names for the {@link Element}.
+     *
+     * @param element the element to check for alternate names.
+     * @return an array of alternate names, or null if there are none.
      */
     @Nullable
     static String[] getAlternateJsonNames(@NotNull Element element) {
-        return (null != element.getAnnotation(SerializedName.class)) ? element.getAnnotation(
-                SerializedName.class).alternate() : null;
+        return null != element.getAnnotation(SerializedName.class) ?
+                element.getAnnotation(SerializedName.class).alternate() :
+                null;
     }
 
+    /**
+     * Creates a {@link TypeSpec} for the the type adapter.
+     *
+     * @param typeTokenConstantsGenerator the generator for the TypeTokenConstants class.
+     * @param stagGenerator               the generator for the Stag class.
+     * @return A non null {@link TypeSpec} describing the adapter class.
+     */
     @NotNull
-    public abstract TypeSpec getTypeAdapterSpec(
-            @NotNull TypeTokenConstantsGenerator typeTokenConstantsGenerator,
-            @NotNull StagGenerator stagGenerator);
+    public abstract TypeSpec createTypeAdapterSpec(@NotNull TypeTokenConstantsGenerator typeTokenConstantsGenerator,
```

Comment:
Yes, this will come in a very remote scenario. I think for now we can just remove this code.